apps/storybook/stories/components/accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,148 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { Accordion as KvibAccordion, AccordionItem as AccItem, AccordionButton, AccordionPanel } from "@kvib/react/src";
+
+const meta: Meta<typeof KvibAccordion> = {
+  title: "Komponenter/Accordion ",
+  component: KvibAccordion,
+  parameters: {
+    docs: {
+      story: { inline: true },
+      canvas: { sourceState: "shown" },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof KvibAccordion>;
+
+export const Accordion: Story = {
+  argTypes: {
+    allowMultiple: {
+      description: "If true, multiple items can be expanded at once.",
+      table: {
+        type: { summary: "boolean" },
+        defaultValue: { summary: false },
+      },
+      control: "boolean",
+    },
+    allowToggle: {
+      description: "If true, expanded items may be collapsed again.",
+      table: {
+        type: { summary: "boolean" },
+        defaultValue: { summary: false },
+      },
+      control: "boolean",
+    },
+    defaultIndex: {
+      description: "Sets the initial index for the expanded accordion item.",
+      table: {
+        type: { summary: "string" },
+      },
+      control: "text",
+    },
+    index: {
+      description: "Sets the index for the expanded accordion item.",
+      table: {
+        type: { summary: "string" },
+      },
+      control: "text",
+    },
+    onChange: {
+      description: "Callback for when the accordion expands/collapses.",
+      table: {
+        type: { summary: "function" },
+      },
+      control: "boolean",
+    },
+    reduceMotion: {
+      description: "If true, disables height animation and transition.",
+      table: {
+        type: { summary: "boolean" },
+        defaultValue: { summary: false },
+      },
+      control: "boolean",
+    },
+  },
+  args: { allowMultiple: true, allowToggle: false, onChange: undefined },
+  render: (args) => (
+    <KvibAccordion {...args}>
+      <AccItem>
+        <AccordionButton>Trykk her</AccordionButton>
+        <AccordionPanel>Tekst tekst tekst tekst tekst tekst tekst tekst tekst tekst tekst</AccordionPanel>
```

Comment:
Jeg syns det er gøy å legge til litt kartverket tekst som eksempler, istedenfor "tekst tekst tekst". Ikke viktig, men syns det blir mer gjennomført😎 Se for eksempel på `Card` og `List`

Reply:
Det var en veldig ide 😄.